Formula/f/faircamp.rb

```diff
@@ -0,0 +1,30 @@
+class Faircamp < Formula
+  desc "Static site generator for audio producers"
+  homepage "https://codeberg.org/simonrepp/faircamp"
+  url "https://codeberg.org/simonrepp/faircamp/archive/0.9.2.tar.gz"
+  sha256 "15c826e22d8297223ae6c591d14ff79226682ef4db3911a1f349d639406295b7"
+  license "AGPL-3.0-or-later"
+
+  depends_on "ffmpeg" => :build
+  depends_on "opus" => :build
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on :macos # Linux builds aren't working at present
```

Comment:
is there any upstream issue for this?

Reply:
Some discussion [here](https://codeberg.org/simonrepp/faircamp/issues/45#issuecomment-1326234) I can elaborate in the comment if desired?

Reply:
Added some more detail [here](https://github.com/Homebrew/homebrew-core/pull/153696/commits/3fe0756c943883e65dabbe960373a9c22a18cf09), I'm a little short on bandwidth right now to sort out the linux side. Is it required? The main interest in `brew` is for the macOS users as linux is maintained by the project author.